dll/apisets/api-ms-win-core-winrt-l1-1-0.spec

```diff
@@ -1,12 +1,12 @@
 
 # This file is autogenerated by update.py
```

Comment:
This script does not need modification in this case?

Reply:
Short: no. Long: ask Mark, who is the author.